02-main/ch2_analysis.tex

```diff
@@ -14,7 +14,7 @@ \section{Tables}
 
 This section contains an examples of table: \autoref{tab:esempio}
 
-\begin{table}[H]
+\begin{table}[!ht]
```

Comment:
I think that's indeed better as an example. I guess overleaf doesn't support absolute specifiers?

Are we really forced to use `!`?

Reply:
I think the ! is not necessary yes, but i've had a better experience when i put it in. It does not change much in most cases but in some it's helped me have a more consistent layout.
I think yeah the absolutes are a bit hard to put into overleaf